packages/embed/src/index.ts

```diff
@@ -266,9 +266,10 @@ export function setup(setupConfig: SetupConfig) {
             ]),
           })
         })
-        .catch(() => {
+        .catch((error) => {
           dispatch({
             type: 'beforeTransactionError',
+            data: { beforeTransactionError: error },
```

Comment:
This should be just the payload as the event name is already defined.

<img width="336" alt="Screenshot 2024-06-01 at 10 56 40 AM" src="https://github.com/gr4vy/gr4vy-embed/assets/1008377/d18a9d6d-1ce9-41b5-9c2b-610c63bc4b24">


```suggestion
            data:  error,
```

Reply:
After looking at the Embed UI PR I can see why you have done it this way - I think this is okay considering the outcome we need :-)